app/views/shared/_run_table.slim

```diff
@@ -38,11 +38,13 @@
               td.align-right.nowrap
                 - if race.in_progress?
                   - now = Time.now.utc
-                  .text-warning data={ms: (now - race.created_at) * 1000}
-                    = Duration.new((now - race.created_at) * 1000).format
+                  .text-warning data={ms: (now - race.started_at) * 1000}
+                    = Duration.new((now - race.started_at) * 1000).format
+                - elsif race.finished?
+                  .text-success Finished
                 - else
                   - if race.public_visibility?
-                    .text-success Public
+                    .text-success Open
```

Comment:
My concern with using `Open` for the visibility is that users might think that a race that is actually in progress is still join-able.  Do you think that that it's clear enough to not be a worry?

Reply:
Yeah I agree, there is an `if` above this that catches in-progress and finished races, so this only says "Open" if the race is public entry *and* not yet started.

Reply:
Woops, must've missed that.